drivers/usb/usbxhci/usbxhci.c

```diff
@@ -97,21 +139,21 @@ XHCI_ResetController(IN PXHCI_HC_OPER_REGS OperRegisters)
      * 25ms should suffice.
      */
     KeQuerySystemTime(&EndTime);
-    EndTime.QuadPart += 25 * 10000;     // 25ms
-    while (TRUE)
+    EndTime.QuadPart += XHCI_POLL_TIME_SET(25);
+    IsPolling = TRUE;
+
+    (VOID)Status;
```

Comment:
This line looks like a cryptic way to write UNREFERENCED_PARAMETER(Status);
Why not deleting that variable entirely instead, including its declaration?
In case you would need it later, the chance is lower that you would forget to remove this left-over-thingy then.

Reply:
Whoops, somehow forgot to remove that. That is not supposed to be there.